appveyor.yml

```diff
@@ -1,74 +0,0 @@
-#
-# Most of these settings are taken from here:
-# https://github.com/ogrisel/python-appveyor-demo/blob/master/appveyor.yml
-#
-
-version: 1.0.{build}
-
-build: off
-
-cache:
-  - '%LOCALAPPDATA%\pip\Cache'
-  - memcached -> appveyor.yml
-
-environment:
-  global:
-    # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
-    # /E:ON and /V:ON options are not enabled in the batch script intepreter
-    # See: http://stackoverflow.com/a/13751649/163740
-    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\.ci\\run_with_env.cmd"
```

Comment:
`run_with_env.cmd` doesn't seem to be used any more. Delete?

Reply:
Sure.

Reply:
P.S., I did try to look for this file.